game/runtime.cpp

```diff
@@ -121,6 +124,14 @@ void ee_runner(SystemThreadInterface& iface) {
                   MAP_ANONYMOUS | MAP_PRIVATE, 0, 0);
   }
 
+  #elif __APPLE__
```

Comment:
good callout by codacy here -- maybe this is your memory issue?

Reply:
The build hasn't quite made it to the runtime yet, this issue occurs within the compiler. I don't think Codacy caught any similar typo in the compiler, while it should prevent issues down the line which is great it won't solve this particular build issue. I haven't even checked to see if I define <sys/mman.h> in the compiler yet so I'll take a look at that next.

Reply:
I think we should try to avoid having an apply specific thing here. Mac OS is a unix, it should support `mmap`.